src/main/java/org/jitsi/jigasi/GatewaySession.java

```diff
@@ -168,6 +184,12 @@ public GatewaySession(SipGateway gateway)
             .getAccountPropertyString(
                 JITSI_MEET_ROOM_PASS_HEADER_PROPERTY,
                 JITSI_MEET_ROOM_PASS_HEADER);
+
+        // check for custom header name for domain base header
+        JITSI_MEET_DOMAIN_BASE_HEADER = sipProvider.getAccountID()
```

Comment:
This is hard to understand, because it's a field and a default value at the same time.

It's an interesting idea, but I don't think we use this anywhere else in our Java code.